webClient/src/app/shared/dialog/overwrite-dataset/overwrite-dataset.component.html

```diff
@@ -11,11 +11,12 @@
 <zlux-tab-trap></zlux-tab-trap>
 <h2 mat-dialog-title>E-tag Mismatch</h2>
 <mat-dialog-content>
-    The dataset {{datasetName}} has been previously updated on the server. Would you like to overwrite the dataset? 
+    The dataset {{datasetName}} has been previously updated on the server. <br>To write, read the dataset again and resolve the difference. <br>Or compare/overwrite the dataset.
```

Comment:
Good looking pop-up but I think it's a bit more wordy than it needs to be. Instead of,

![image](https://user-images.githubusercontent.com/20528015/141225646-3e7a29fb-99b8-43db-8f4d-ce0baa90d108.png)

let's do,

![image](https://user-images.githubusercontent.com/20528015/141226129-106a45e9-6c34-4ad7-88c8-518c26f94c9c.png)